### PR TITLE
10 create user skill table

### LINKED ISSUE #21: 10 - Create table User-Skill
- [x] Create a junction table for User-skill

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
creating the junction table for user skill
- user-skill.js
- create-user-skill.js
and adding user-skill file to sync.js file

### CODE REVIEW

File: api/db/scripts/create-user-skill.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+         {
+            UserSillID: 3,
+            UserID: 3,
+            SkillID: 5,
+            Level: 3,
+            SelectedSkill: 3
+        }
+    ]);
+    
+    try {
```
Comment (rayanejsilva) [inline api/db/scripts/create-user-skill.js]: Can you delete all try and catch before you push the files?

**Review by rayanejsilva — CHANGES_REQUESTED** [review]
In the user.js you need to require user-skill and change through attribute as a model

**Review by Hajir-pro — APPROVED** [review]

File: api/db/scripts/create-user-skill.js (hunk 1 above)
Comment (Moamin-AbuEwaida) [reply api/db/scripts/create-user-skill.js]: yea @rayanejsilva 
I'll do that and re push the PR

Comment (Moamin-AbuEwaida) [reply api/db/scripts/create-user-skill.js]: just repushed the changes @rayanejsilva

**Review by rayanejsilva — APPROVED** [review]

**Comment by rayanejsilva** [comment]
LGTM!

**Pull request merged**